src/renderer/pages/project/script_editor/styles/chara_params.vue

```
@@ -0,0 +1,48 @@
+</template>
+
+<script setup lang="ts">
+import { Checkbox, Card } from "@/components/ui";
+import { type MulmoImageParamsImages } from "mulmocast/browser";
+import { useI18n } from "vue-i18n";
+
+const { t } = useI18n();
+
+interface Props {
+  images?: MulmoImageParamsImages;
+  beat: MulmoBeat;
```

ESLint

[error] 'MulmoBeat' is not defined.

(no-undef)